dll/win32/comctl32/commctrl.c

```diff
@@ -686,7 +686,11 @@ void WINAPI DrawStatusTextW (HDC hdc, LPCRECT lprc, LPCWSTR text, UINT style)
         border = 0;
 
     oldbkcolor = SetBkColor (hdc, comctl32_color.clrBtnFace);
+#ifdef __REACTOS__
```

Comment:
As what is happening is not fully "explained":
```suggestion
#ifdef __REACTOS__ // HACK for CORE-19854.
```

Reply:
Done. Thanks.